crypto/cmp/cmp_msg.c

```diff
@@ -817,8 +817,13 @@ OSSL_CMP_MSG *ossl_cmp_certConf_new(OSSL_CMP_CTX *ctx, int fail_info,
     if ((certStatus = OSSL_CMP_CERTSTATUS_new()) == NULL)
         goto err;
     /* consume certStatus into msg right away so it gets deallocated with msg */
-    if (!sk_OSSL_CMP_CERTSTATUS_push(msg->body->value.certConf, certStatus))
+    if (sk_OSSL_CMP_CERTSTATUS_push(msg->body->value.certConf, certStatus) < 1) {
+    /*
+     * It hasn't been consumed by the msg at this point of time,
+     * so we should free certStatus */
```

Comment:
move */ onto next line..
````
/*
 *
 *
 */
````

Reply:
> move */ onto next line..
> 
> ```
> /*
>  *
>  *
>  */
> ```

I fixed it :ok_hand:

Reply:
Actually I agree with https://github.com/openssl/openssl/issues/20145#issuecomment-1441768313 that the whole comment is not needed.
As the [coding guidelines](https://www.openssl.org/policies/technical/coding-style.html) say: 

> Comments are good, but there is also a danger of over-commenting.

so don't comment the obvious.

Reply:
If I replaced it by : "not consumed by the msg at this point of time", is it good ? Or I completely remove it ?

Reply:
As I wrote in my below request for changes, I propose removing the comment entirely.

If we consistently placed comments like this, much of the OpenSSL code would become pretty cluttered.